resources/ext/ezbake.conf

```diff
@@ -20,22 +19,42 @@ ezbake: {
                preinst: ["install --group={{user}} --owner={{user}} -d /var/lib/puppet/jruby-gems",
                          "mkdir -p /etc/puppet/manifests"],
```

Comment:
@haus just noticed this... `/etc/puppet/manifests` is probably no longer the right directory, right?